source/gettingStarted/101.md

```diff
@@ -13,19 +13,19 @@ The key to working on projects at Apache (and on any open source project, for th
 is to have a personal reason for being involved. You might be trying to solve 
 a day job issue, you might be looking to learn a new technology or you might 
 simply want to do something fun in your free time. We don't care what your 
-motivation is we just care about you wanting to get involved.
+motivation is; we just care about you wanting to get involved.
 
-If you don't have a specific technical issue to solve you might be willing to 
+If you don't have a specific technical issue to solve, you might be willing to 
 work on any project. Our [projects page][1] provides a useful index which 
-allows you to view projects alphabetically, by category or by coding language. When you view a projects detail page in this list you will find information about their 
+allows you to view projects alphabetically, by category or by coding language. When you view a project's details page in this list you will find information about their 
 mailing lists, issue tracker and other resources.
 
 You might also like to view our 
 [Incubating projects][2]. These projects work in exactly the same way as our "top
 level projects" but are still developing their initial community.
 
-Once you've found some interesting-looking projects, join their mailing lists and
-start "lurking". Read the mails that come through the list. Initially you will not
+Once you've found some interesting-looking projects, join their email lists and
```

Comment:
Do we want to update `mailing list` to `email lists`? Normally `mailing list` is the general term used throughout the ASF, right? What do you think @bdelacretaz ?

Reply:
+1, will be good to use the `mailing lists` term to maintain the consistency.

Reply:
The term "mailing lists" is consistently used across the entire open source ecosystem, and is well understood. I don't think it needs to be a huge debate, but I do prefer to retain "mailing list" as our canonical term.